apps/explorer/vitest.config.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
You shouldn't need a separate vitest config, you should be able to use the base vite config.

Reply:
It does not recognise the `vitest`  test property even when include `// <reference types="vitest" />`

Reply:
The only way I got it to work is to switch to `import { defineConfig } from 'vitest/config'` even with that plugins is not typed properly